tests/mock_tables/LLDP_ENTRY_TABLE.json

```diff
@@ -49,7 +49,7 @@
     "lldp_rem_chassis_id": "00:11:22:33:44:55",
     "lldp_rem_sys_desc": "I'm a little teapot.",
     "lldp_rem_time_mark": 18545,
-    "lldp_rem_port_desc": "this is a port description",
```

Comment:
>this is a port description", [](start = 27, length = 28)

Why change test data?

Reply:
This question is still applicable

---
In reply to: [526610376](https://github.com/Azure/sonic-dbsyncd/pull/32#discussion_r526610376) [](ancestors = 526610376)

Reply:
Sorry @qiluo-msft forgot to respond. 
Yes this change was needed as mention in description there is bug with unit test test_sync_roundtrip() where LLDP table information for eth0 and Ethernet0 can overwrite each since testcase was making dictionary with key based on integer part of port name. 
I have fixed this now to use complete string as comparison and with the fix needed to update test data to match expectation

Below code data for eth0 and Ethernet0 can overwrite each other:
  
>    # convert dict keys to ints for easy comparison	        # convert dict keys to ints for easy comparison
>         jo = {int(re.findall(r'\d+', k)[0]): v for k, v in parsed_update.items()}	        jo = {'LLDP_ENTRY_TABLE:'+ k: v for k, v in parsed_update.items()}
>         r_out = {int(re.findall(r'\d+', k)[0]): v for k, v in dump.items()}

Reply:
@qiluo-msft Updated the expected data and reverted test data.